src/index.js

```diff
@@ -0,0 +1,29 @@
+import express from 'express';
+import dotenv from 'dotenv';
+import cors from 'cors';
+import productRoutes from './routes/productRoutes.js';
+import articleRoutes from './routes/articleRoutes.js';
+import commentRoutes from './routes/commentRoutes.js';
+import { error } from './middlewares/error.js';
+
+dotenv.config();
+
+const app = express();
+const PORT = process.env.PORT || 4000;
+
+app.use(cors());
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+
+app.use('/uploads', express.static('uploads'));
+
+app.use('/products', productRoutes);
+app.use('/articles', articleRoutes);
+app.use('/', commentRoutes);
```

Comment:
comment router에도 경로를 추가해주면 좋을 것 같습니다,.
```suggestion
app.use('/comment', commentRoutes);
```